### ISSUE TITLE
APIDocumentation_LoadUI error referencing removed MAX_STABLE_SLOTS

### ISSUE THREAD

**Issue by mbattersby** [Stanzilla/WoWUIBugs#569]
`/api list`

Error in `Interface/AddOns/Blizzard_APIDocumentationGenerated/PetConstantsDocumentation.lua`

```
Message: ...DocumentationGenerated/PetConstantsDocumentation.lua:14: attempt to perform arithmetic on global 'MAX_STABLE_SLOTS' (a nil value)
Time: Fri May 10 13:59:58 2024
Count: 1
Stack: ...DocumentationGenerated/PetConstantsDocumentation.lua:14: attempt to perform arithmetic on global 'MAX_STABLE_SLOTS' (a nil value)
[string "@Interface/AddOns/Blizzard_APIDocumentationGenerated/PetConstantsDocumentation.lua"]:14: in main chunk
[string "=[C]"]: ?
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface/AddOns/Blizzard_UIParent/Mainline/UIParent.lua"]:402: in function `UIParentLoadAddOn'
[string "@Interface/AddOns/Blizzard_UIParent/Mainline/UIParent.lua"]:619: in function `APIDocumentation_LoadUI'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:2606: in function `?'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5422: in function `ChatEdit_ParseText'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5074: in function `ChatEdit_SendText'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5110: in function `ChatEdit_OnEnterPressed'
[string "*ChatFrame.xml:140_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:140_OnEnterPressed"]:1>

Locals: (*temporary) = <table> {
}
(*temporary) = <table> {
}
(*temporary) = <table> {
 Type = "Constants"
 Name = "PetConsts"
}
(*temporary) = <table> {
}
(*temporary) = <table> {
 Value = 200
 Type = "number"
 Name = "MAX_STABLE_SLOTS"
}
(*temporary) = <table> {
 Value = 25
 Type = "number"
 Name = "MAX_SUMMONABLE_PETS"
}
(*temporary) = <table> {
 Value = 5
 Type = "number"
 Name = "MAX_SUMMONABLE_HUNTER_PETS"
}
(*temporary) = <table> {
 Value = 5
 Type = "number"
 Name = "NUM_PET_SLOTS_THAT_NEED_LEARNED_SPELL"
}
(*temporary) = <table> {
 Type = "number"
 Name = "NUM_PET_SLOTS"
}
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to perform arithmetic on global 'MAX_STABLE_SLOTS' (a nil value)"
```

**Comment by Meorawr** [comment]
Worth noting that this has been fixed for a number of builds in 11.0 already. I suspect the odds are low of this fix being backported as it's not got a lot of impact.

**Comment by Meorawr** [comment]
As prepatch looms closer, will just close as any backport is almost certainly not happening at this point.
